Bottom: 66f2a0e72b2d1fb848bca94b10bae085d8e79058
Top:    c82596dbbebc8cf6633f7d5c88016cff86417b3d
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-05 09:37:40 -0500

Refresh of fixup-tmp-rework-ip-address-network

---
diff --git a/roles/docker-networking/tasks/main.yml b/roles/docker-networking/tasks/main.yml
index 1b1569c6..bc56ac58 100644
--- a/roles/docker-networking/tasks/main.yml
+++ b/roles/docker-networking/tasks/main.yml
@@ -3,7 +3,7 @@
 
 - assert:
     that:
-    - net_connection is defined
+    - network_connection is defined
     - subnet is defined
     - services is defined
